components/dialog/index.css

```diff
@@ -48,6 +48,16 @@ governing permissions and limitations under the License.
 
 	/* Hero */
 	--spectrum-standard-dialog-hero-block-size: 140px;
+
+	/* Fullscreen/fullscreenTakeover */
+	--spectrum-takeover-dialog-title-font-size: var(--spectrum-title-size-xxl);
+	--spectrum-takeover-dialog-spacing-grid-padding: var(--spectrum-spacing-500);
+	--spectrum-takeover-dialog-spacing-header-gap: var(--spectrum-spacing-400);
+	--spectrum-takeover-dialog-spacing-title-to-body: var(--spectrum-spacing-500);
+	--spectrum-takeover-dialog-inline-size: var(--spectrum-takeover-dialog-width);
+
+	/* TODO: takeover-dialog-height token needs to be updated to 100% before we can use the custom property. */
```

Comment:
This is taken care and the custom property and token are in use. Remove this comment.